python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -144,6 +160,34 @@ def _get_pre_sensor_auto_materialize_cursor(
         return deserialize_value(serialized_cursor, AssetDaemonCursor)
 
 
+def load_asset_scoped_cursors(
+    asset_keys: AbstractSet[AssetKey], instance: DagsterInstance
+) -> Dict[AssetKey, str]:
+    """Load the cursors for a set of asset keys from the daemon cursor storage, returning a dictionary
+    of the cursors keyed by asset key.
+    """
+    cursor_values = instance.daemon_cursor_storage.get_cursor_values(
+        {asset_key_to_cursor_key(asset_key) for asset_key in asset_keys}
+    )
+    return {
+        cursor_key_to_asset_key(asset_key): cursor for asset_key, cursor in cursor_values.items()
+    }
+
+
+# This prefix is affixed to the front of a serialized AssetKey to create the key used to store its
+# evaluation state in the daemon cursor storage.
+_ASSET_CURSOR_PREFIX: Final = "ASSET_CURSOR__"
```

Comment:
nit-picky, but maybe let's make the value `"SCHEDULING_CURSOR__"` which leaves open options for scheduling non-asset entities (e.g. checks) in the future